include/leveldb/perf_count.h

```diff
@@ -240,6 +240,9 @@ enum PerformanceCountersEnum
     ePerfExpiredKeys=94,      //!< key physically removed because it expired
     ePerfExpiredFiles=95,     //!< entire file removed because all keys expired
 
+    ePerfSyslogWrite=96,      //!< logged message to syslog
+    ePerfBackupStarted=97,    //!< hot backup initiated
```

Comment:
only need one or the other ... and Matthew is only known person to look at these values.  Adding error.